### PR TITLE
Move assets to /usr/share/v2ray

### LINKED ISSUE #142: can you move the asset to /usr/share/v2ray/ ?
Please attach the output of following commands.

```
cat /etc/*release
DISTRIB_ID='OpenWrt'
DISTRIB_RELEASE='21.02.0'
DISTRIB_REVISION='r16279-5cc0535800'
DISTRIB_TARGET='bcm27xx/bcm2711'
DISTRIB_ARCH='aarch64_cortex-a72'
DISTRIB_DESCRIPTION='OpenWrt 21.02.0 r16279-5cc0535800'
DISTRIB_TAINTS=''
NAME="OpenWrt"
VERSION="21.02.0"
ID="openwrt"
ID_LIKE="lede openwrt"
PRETTY_NAME="OpenWrt 21.02.0"
VERSION_ID="21.02.0"
HOME_URL="https://forum.openwrt.org/t/rpi4-community-build/69998"
BUG_URL="https://forum.openwrt.org/t/rpi4-community-build/69998"
SUPPORT_URL="https://forum.openwrt.org/t/rpi4-community-build/69998"
BUILD_ID="r16279-5cc0535800"
OPENWRT_BOARD="bcm27xx/bcm2711"
OPENWRT_ARCH="aarch64_cortex-a72"
OPENWRT_TAINTS=""
OPENWRT_DEVICE_MANUFACTURER="rpi-foundation"
OPENWRT_DEVICE_MANUFACTURER_URL="https://www.raspberrypi.org"
OPENWRT_DEVICE_PRODUCT="Generic"
OPENWRT_DEVICE_REVISION="v0"
OPENWRT_RELEASE="OpenWrt 21.02.0 r16279-5cc0535800"
localversion="1.0.3-2"
opkg print-architecture
arch all 1
arch noarch 1
arch aarch64_cortex-a72 10
```
referring to default [asset](https://github.com/v2fly/v2ray-core/blob/4a174e8aa66ce0aaccba602e87a032e101fea315/common/platform/others.go#L35) can you update the script?

### PR DESCRIPTION
**Pull request by mzz2017**
Fix #142

### CODE REVIEW

**Comment by kuoruan** [comment]
Hi @mzz2017 Thank you for the PR. But I do not plan to move the asset files to v2ray.
The asset files should put the same folder as v2ray and v2ctl. So that v2ray can find it when the config file contains some string like `geoip:private`
If you need this, you can move it manually and set a env variable `V2RAY_LOCATION_ASSET`
https://www.v2ray.com/en/configuration/env.html#asset

**Comment by mzz2017** [comment]
This docs is too old. And place the asset in the same directory with v2ray binary is not recommended.

See the latest code https://github.com/v2fly/v2ray-core/blob/deb9d08487501e73c6693fb0ac90bd91952cb1a6/common/platform/others.go#L34

File: Makefile [hunk 1]
```diff
@@ -331,7 +331,7 @@ define Package/v2ray-core/install
```
Comment (kuoruan) [inline Makefile:331]: Add `$(INSTALL_DIR) $(1)/usr/share`

Comment (mzz2017) [reply Makefile:331]: Done

**Pull request merged**